world.js

```diff
@@ -833,11 +880,11 @@ world.addEventListener('trackedobjectadd', async e => {
   const trackedObjectJson = trackedObject.toJSON();
   const {instanceId, parentId, contentId, position, quaternion, options: optionsString} = trackedObjectJson;
   const options = JSON.parse(optionsString);
+  const res = await fetch(`https://tokens.webaverse.com/${contentId}`);
+  const token = await res.json();
 
   const file = await (async () => {
-    if (typeof contentId === 'number') {
-      const res = await fetch(`https://tokens.webaverse.com/${contentId}`);
-      const token = await res.json();
+    if (typeof contentId === 'number' || /^\d+$/.test(contentId)) {
```

Comment:
If `contentId` comes in as a string that is probably a bug elsewhere.

Reply:
f479cbe